src/test/lrsql/admin/protocol_test.clj

```diff
@@ -130,3 +149,33 @@
           (is (= []
                  (adp/-get-api-keys lrs acc-id))))))
     (component/stop sys')))
+
+(deftest status-test
+  (let [sys  (support/test-system)
+        sys' (component/start sys)
+        lrs  (:lrs sys')]
+    (testing "Get LRS status"
+      (is (= {:statement-count       0
+              :actor-count           0
+              :last-statement-stored nil
+              :platform-frequency    {}
+              :timeline              []}
+             (adp/-get-status lrs {})))
+      ;; add a statement
+      (lrsp/-store-statements lrs auth-ident [stmt-0] [])
+      (let [last-stored (get-in
+                         (lrsp/-get-statements lrs auth-ident {} [])
+                         [:statement-result
+                          :statements
+                          0
+                          "stored"])]
+        (is (= {:statement-count       1
```

Comment:
I also feel like it would be helpful to have a test on more than one statement. For example, the second statement could share the same actor or platform as the previous, demonstrating that the values are not coupled with each other.

Reply:
I agree, BUT: Given the nature of the timeline query doing this with 2 statements technically introduces a false test fail that could happen (at the very least) once per year. 😸 
I'll see what I can do.

Reply:
Why would that happen? Daylights Savings Time?

Reply:
If the unit is the default (day), you could technically have a case where the two stamps could have differing days.

Reply:
What I do here should prevent it from ever happening: https://github.com/yetanalytics/lrsql/pull/282/commits/9f3dd96843e62bb1756dfdfded6a242e0c7e79ac